Packs/Cloudflare_New/Integrations/CloudflareNew/CloudflareNew.py

```diff
@@ -0,0 +1,250 @@
+        zone_names = argToList(websites["name"])
+
+    # for i, zone_id in enumerate(zone_ids):
+    for i in range(len(zone_ids)):
+        query = """
+        query ListFirewallEvents($zoneTag: String, $filter: FirewallEventsAdaptiveFilter_InputObject) {
+            viewer {
+            zones(filter: { zoneTag: $zoneTag }) {
+                firewallEventsAdaptive(
+                filter: $filter
+                limit: 100
+                orderBy: [datetime_DESC]
+                ) {
+                action
+                clientAsn
+                clientCountryName
+                clientIP
+                clientRequestPath
+                clientRequestQuery
+                datetime
+                source
+                userAgent
+                }
+            }
+            }
+        }
+        """
+
+        payload = {
+            "query": query,
+            "variables": {
+                "zoneTag": zone_ids[i],
+                "filter": {
+                    "datetime_geq": start_time,    # .strftime("%Y-%m-%dT%H:%M:%SZ")
+                    "datetime_leq": end_time,    # .strftime("%Y-%m-%dT%H:%M:%SZ")
+                    "clientIP": source_ip
+                }
+            }
+        }
+
+        headers = {} if headers is None else headers
+        response = client.http_request("POST", url_suffix="/v4/graphql", headers=headers, data=payload)
+
+        if response.status_code == 200:
+            cloudflare_events = response.json().get("data")["viewer"]
+            act_result["cloudflare_events"] = cloudflare_events
+
+            return_results(CommandResults(outputs=act_result, readable_output=act_result))
+
+            human_readable = tableToMarkdown(
+                "Query " + zone_names[i].upper() + " Logs:",
+                act_result["cloudflare_events"]["zones"][0]["firewallEventsAdaptive"],
+                ["action", "clientAsn", "clientCountryName", "clientIP", "clientRequestPath",
+                    "clientRequestQuery", "datetime", "source", "userAgent"],
+                removeNull=True,
+            )
+
+            # data_to_table = parse_list_to_table(act_result["cloudflare_events"]["zones"][0]["firewallEventsAdaptive"])
+            # return_results(
+            #     "Type": entryTypes["note"],
+            #     "Contents": data_to_table,
+            #     "ContentsFormat": formats["table"]
+            # )
+
+            return_results(
+                {
+                    "Type": entryTypes["note"],
+                    "ContentsFormat": formats["json"],
+                    "Contents": act_result,
+                    "ReadableContentsFormat": formats["markdown"],
+                    "HumanReadable": human_readable,
+                    "IgnoreAutoExtract": False
+                }
+            )
+
+            if not cloudflare_events['zones'][0]['firewallEventsAdaptive']:
+                continue
+            print("Zone_name: {}".format(zone_names[i]), zone_id)
```

Comment:
please remove usage of print, you can replace it with demisto.debug() or demisto.info()